portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String textValue = request.getParameter("text-input");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("You submitted the email address: " + textValue);
+
+    // Write the value to the response so the user can see it.
+    response.getWriter().println("You submitted the email address: " + textValue);
```

Comment:
The text here is written twice, and it's also the same pattern of code used below:

```
System.out.println(something);
response.getWriter().println(something);
```

That makes it a great place to make a helper function:

```
private void writeToResponse(HttpServletResponse response, String text) {
    System.out.println(text);
    response.getWriter().println(text);
}
```